packages/core/src/models/user_importers.ts

```
@@ -0,0 +1,46 @@
+  extra?: T;
+  source: UserImportSource;
+};
+
+export type ImportPrivyUserOutput = ImportUserOutput<PrivyUser>;
+
+export type ImportContractUserOutput = ImportUserOutput<ContractUser>;
+
+export type ImportedUserSourceData =
+  | {
+      source: UserImportSource;
+      sourceData: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)